### ISSUE TITLE
angular-translate with params

### ISSUE THREAD

**Issue by kasperlewau** [kasperlewau/angular-bind-notifier#3]
Currently the following does not work: 

`{{:namespace:'string_to_translate' | translate: { someValue: '...' }}}`

due to the object literal notation of the translate filter and the [splitting regex](https://github.com/kasperlewau/angular-bind-notifier/blob/master/src/bindNotifier.js#L95) in angular-bind-notifier.

**Comment by kasperlewau** [comment]
Current workaround: 

```html
<span translate=":namespace:'string_to_translate'" translate-values="{ someValue: '...'}"></span>
```

**Comment by kasperlewau** [comment]
partially fixed in https://github.com/kasperlewau/angular-bind-notifier/commit/f0a187f53bcfefff4781696b4da7c3a14cd51efa. 

As long as a `:` is immediately followed by a whitespace character, the notifier/expression split wont catch - and the expression won't break into smaller chunks so as to be regarded a notifier. 

Will keep open for the time being, not 100% *yet.*

**Comment by kasperlewau** [comment]
Closed by #8
